client/rhel/yum-rhn-plugin/po/ca.po

```diff
@@ -90,9 +90,9 @@ msgstr "Missatge d'error:"
 
 #: ../rhnplugin.py:276
 msgid "Please run rhn_register as root on this client"
-msgstr "Si us plau, executeu rhn_register com a root sobre aquest client"
+msgstr "Si us plau, executeu rhn_register com a arrel en aquest client."
```

Comment:
I am not catalonian speaker myself, but I am pretty sure catalonians do not translate the `root` user into `arrel` user. https://ca.wikipedia.org/wiki/Superusuari

@srbarrios can you have a quick look at the catalonian translations of this PR? Maybe there are some other typos.